package.json

```diff
@@ -41,7 +41,7 @@
     "jsdoc": "^3.3.2",
     "karma": "6.3.4",
     "karma-chrome-launcher": "3.1.0",
-    "karma-firefox-launcher": "2.1.0",
+    "karma-firefox-launcher": "2.1.1",
     "karma-cli": "2.0.0",
```

Comment:
is this change from you?

Reply:
I don't think so.

Reply:
@jvigliotta @kobe1104 that was me. I was trying to bust the cache

Reply:
FINE

Reply:
Oh I get it...

Reply:
still seems weird that it would be a diff in this PR and master

Reply:
Hi @unlikelyzero do we need to change the karma-firefox-launcher version back?

Reply:
@davetsay @jvigliotta I've confirmed with @unlikelyzero that the change of karma-firefox-launcher version won't cause an issue. I'll leave it as is.